packages/firestore/src/model/document_key.ts

```diff
@@ -1,6 +1,6 @@
 /**
  * @license
- * Copyright 2017 Google LLC
+ * Copyright 2020 Google LLC
```

Comment:
If you change this back to 2017 and add a newline back after `import { debugAssert }` this file we be unchanged in this PR.

Reply:
Done